backend/app/views/spree/admin/customer_returns/_reimbursements_table.html.erb

```diff
@@ -21,7 +21,7 @@
         <td><%= reimbursement.display_total %></td>
         <td>
           <span class="state <%= reimbursement_status_color(reimbursement) %>">
-            <%= reimbursement.reimbursement_status %>
+            <%= t 'spree.'+reimbursement.reimbursement_status %>
```

Comment:
I also looked into documentation: https://guides.spreecommerce.com/developer/i18n.html

It's better to use `Spree.t :name` instead of `t :name, scope: :spree`